projects/storefrontlib/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -40,8 +42,22 @@ import { map, startWith } from 'rxjs/operators';
 export class FormErrorsComponent implements DoCheck {
   private featureConfigService = inject(FeatureConfigService);
 
+  private elementRef = inject(ElementRef);
+
   constructor(protected ChangeDetectionRef: ChangeDetectorRef) {
     useFeatureStyles('a11yFormErrorMuteIcon');
+    if (this.featureConfigService.isEnabled('a11yImprovedErrorMessage')) {
+      const ariaLive = inject(new HostAttributeToken('aria-live'), {
+        optional: true,
+      });
+
+      if (!ariaLive) {
+        // If no aria-live value is set add 'polite' as a default. This is preferred over setting
+        // role='alert' so that screen readers do not interrupt the current task to read this aloud.
+        this.elementRef.nativeElement.setAttribute('aria-live', 'polite');
```

Comment:
We should be careful with DOM manipulation on page instantiation because it can cause issues with SSR and hydration. Is it possible to set the attribute using an Angular-friendly method (eg. template, input value)?
https://angular.dev/guide/hydration#direct-dom-manipulation